woocommerce/woocommerce-ios
# Remove BottomSheet copy from WCiOS and use it from WordPressUI framework

Fixes #2054 

## Changes

- Removed our own copy of bottom sheet code, and imported `WordPressUI` in its usage instead
- In `BottomSheetListSelectorViewController`, updated its `preferredContentSize` to be the calculated `contentSize` without additional top margin from later bottom sheet changes in the framework

## Testing

- Go to the Products tab
- Tap on the "Sort by" CTA under the WIP banner --> the bottom sheet or popover should appear as expected
- Tap on a simple product with missing information in inventory/shipping/short description
- Tap on the "Add more details" CTA at the bottom --> the bottom sheet or popover should appear as expected

## Example screenshots

Since we started reusing the bottom sheet framework from `WordPressUI` in this PR, let's sanity check on iPad and iPhone:

landscape | portrait
-- | --
![Simulator Screen Shot - iPad Air 2 - 2020-05-14 at 12 38 16](https://user-images.githubusercontent.com/1945542/81894199-cd229f80-95e1-11ea-87a9-6e59a8fe62b8.png) | ![Simulator Screen Shot - iPad Air 2 - 2020-05-14 at 12 38 13](https://user-images.githubusercontent.com/1945542/81894188-c8f68200-95e1-11ea-8db9-d6dc96875c7c.png)
![Simulator Screen Shot - iPad Air 2 - 2020-05-14 at 12 49 06](https://user-images.githubusercontent.com/1945542/81894201-ce53cc80-95e1-11ea-9fca-7ad995ee2aef.png) | ![Simulator Screen Shot - iPad Air 2 - 2020-05-14 at 12 49 12](https://user-images.githubusercontent.com/1945542/81894206-d0b62680-95e1-11ea-8c45-4de6c06b7824.png)
![Simulator Screen Shot - iPhone 8 Plus - 2020-05-14 at 12 44 24](https://user-images.githubusercontent.com/1945542/81894210-d3188080-95e1-11ea-803a-632e402b883d.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2020-05-14 at 12 44 25](https://user-images.githubusercontent.com/1945542/81894214-d3b11700-95e1-11ea-8a68-e2d458ad78cc.png)


Update release notes:
